plexapi/alert.py

```diff
@@ -55,7 +55,8 @@ def stop(self):
         log.info('Stopping AlertListener.')
         self._ws.close()
 
-    def _onMessage(self, ws, message):
+    def _onMessage(self, *args):
+        message = args[-1]
```

Comment:
Can you please add a comment here explaining the wonkyness between versions and why we're grabbing the last item? We're all going to forget in about 3 weeks time. :D